### PR TITLE
Fix bugs: fix some bugs in bluestore deployment

### PR DESCRIPTION
**Pull request by tanghaodong25**

### CODE REVIEW

**Comment by tanghaodong25** [comment]
@xuechendi  please take a look.

File: benchmarking/mod/benchmark.py [hunk 1]
```diff
@@ -175,6 +175,7 @@ def run(self):
                     common.pdsh(user, [node], "cd %s; blktrace -d %s -o `hostname`_journal_%s 2>&1 > %s/`hostname`_blktrace_%s.log" % (dest_dir, journal_dev, journal_dev.replace("/","_"), dest_dir, journal_dev.replace("/","_")), option="force")
         if "fatrace" in self.cluster["collector"]:
             common.printout("LOG","Start fatrace data collector under %s " % nodes)
+            time.sleep(15)
```
Comment (xuechendi) [inline benchmarking/mod/benchmark.py]: why sleep here?

File: deploy/mod/deploy.py [hunk 2]
```diff
@@ -37,6 +37,7 @@ def __init__(self, tunings=""):
         self.cluster["ceph_conf"]["global"]["auth_client_required"] = "none"
         self.cluster["ceph_conf"]["mon"]["mon_data"] = "/var/lib/ceph/mon.$id"
         self.cluster["ceph_conf"]["osd"]["osd_data"] = "/var/lib/ceph/mnt/osd-device-$id-data"
+        #self.cluster["ceph_conf"]["osd"]["osd_data"] = "/var/lib/ceph/osd/ceph-$id"
```
Comment (xuechendi) [inline deploy/mod/deploy.py]: delete this line?

File: benchmarking/mod/benchmark.py (hunk 1 above)
Comment (tanghaodong25) [inline benchmarking/mod/benchmark.py]: In my environment, some connection error always occurs if using fatrace without sleep here.

**Pull request merged**